google/cloud/forseti/services/inventory/storage.py

```diff
@@ -750,6 +750,19 @@ def _get_parent_name(asset):
             # Strip off the last two segments of the name to get the parent
             return '/'.join(asset['name'].split('/')[:-2])
 
+        elif asset['asset_type'] in ('k8s.io/Node', 'k8s.io/Namespace'):
+            return '/'.join(asset['name'].split('/')[:-3])
```

Comment:
I recommend adding comments that show what the raw data looks like and what the parent is, so a reviewer can tell if the code is correct (same below).

Reply:
Done.

Reply:
Thanks, that helped me catch a problem with the iterators above. Now please remove the real project IDs from the comments, use project1 or similar. Same with any real node names or other identifiers.

Reply:
Done.